package.json

```diff
@@ -60,6 +61,7 @@
     "@material-ui/core": "^5.0.0-beta.2",
     "@material-ui/icons": "^5.0.0-beta.1",
     "copy-webpack-plugin": "^7.0.0",
+    "humanize-duration": "^3.27.1",
```

Comment:
I'd prefer using `date-fns` instead of this package.
See: https://date-fns.org/v2.28.0/docs/intervalToDuration

Reply:
Done